framework_tvb/tvb/core/services/project_service.py

```diff
@@ -657,8 +653,6 @@ def remove_datatype(self, project_id, datatype_gid, skip_validation=False):
                 ### Do not remove Operation in case DataType still exist referring it.
                 continue
             correct = correct and dao.remove_entity(Operation, operation_id)
-            ## Make sure Operation folder is removed
-            self.structure_helper.remove_operation_data(project.name, datatype.fk_from_operation)
```

Comment:
I also think this line should be put back and not removed.

Reply:
Readded it.